android/app/src/main/java/app/organicmaps/widget/placepage/PlaceDescriptionActivity.java

```diff
@@ -16,10 +16,18 @@ protected Class<? extends Fragment> getFragmentClass()
     return PlaceDescriptionFragment.class;
   }
 
-  public static void start(@NonNull Context context, @NonNull String description)
+  public static void start(@NonNull Context context, @NonNull String title, @NonNull String description)
   {
     Intent intent = new Intent(context, PlaceDescriptionActivity.class)
-        .putExtra(PlaceDescriptionFragment.EXTRA_DESCRIPTION, description);
+        .putExtra(PlaceDescriptionFragment.EXTRA_DESCRIPTION, description)
+        .putExtra("toolbarTitle", title);
```

Comment:
To avoid potential mistakes, it's better to introduce a private static final string variable instead of copypasting "toolbarTitle" manually in two places.